build.sbt

```diff
@@ -585,100 +472,141 @@ def isJdiJar(file: File): Boolean = {
   if (!System.getProperty("java.specification.version").startsWith("1.")) false
   else file.getAbsolutePath.contains(SbtJdiTools.JavaTools.getAbsolutePath.toString)
 }
+val publishJsonModuleSettings = List(
+  publishM2Configuration := publishM2Configuration.value.withOverwrite(true),
+  publishLocalConfiguration := publishLocalConfiguration.value.withOverwrite(true),
+  // We compile in both so that the maven integration can be tested locally
+  publishLocal := publishLocal.dependsOn(publishM2).value
+)
+
+lazy val integrationUtils211 = project
```

Comment:
We could probably remove 2.11 and just note that for Gradle you need to use the last version which is 1.5.4. We can do it later though.

Reply:
Sounds good. Supporting 11 isn't causing any real issues at the moment, but the moment it does yea, we should rip it out.